tests/files/packet_centos7-flannel-containerd-addons-ha.yml

```diff
@@ -23,7 +23,6 @@ cert_manager_enabled: true
 metrics_server_enabled: false
 metrics_server_kubelet_insecure_tls: true
 kube_token_auth: true
-kube_basic_auth: true
```

Comment:
This is linked to https://github.com/kubernetes-sigs/kubespray/pull/6655, as it's no longer available in 1.19 anyhow